app/views/shared/_interactive_map.html.erb

```diff
@@ -1,55 +1,59 @@
-<% if locals[:geojson].present? %>
-  <div class="map">
-    <div id="<%= locals[:div_id] %>" style="width: 680px; height: 400px;"></div>
-    <div id="coordinates"></div>
-  </div>
-
-  <script>
-    var <%= locals[:div_id] %>Map = L.map(<%= locals[:div_id] %>);
-      L.tileLayer('https://api.os.uk/maps/raster/v1/zxy/Road_3857/{z}/{x}/{y}.png?key=OmwuvG9HvHq4qKgm8QwaTtTRCMJ1psvk', {
-      attribution: 'Contains OS data &copy; Crown copyright and database rights 2021 OS (0)100019252',
-      maxZoom: 20
-    }).addTo(<%= locals[:div_id] %>Map);
-
+<div class="map" style="margin-bottom: 1em">
```

Comment:
It's not in the ACs but I'm wondering if it could be an improvement to have the map zoomed to the same resolution as the pre-drawn one? Mine is defaulting to showing a big chunk of the SouthEast - maybe we could zoom it to the postcode plus or minus however many clicks?

Reply:
Yes, I think this is a good idea too, but I didn't want to implement it (requires doing geolookups, which we don't have at the moment), without discussing with product/design to prioritise appropiately.